recipes/mpg123/all/conanfile.py

```diff
@@ -0,0 +1,191 @@
+from conans import AutoToolsBuildEnvironment, CMake, ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+
+class Mpg123Conan(ConanFile):
+    name = "mpg123"
+    description = "Fast console MPEG Audio Player and decoder library"
+    topics = ("conan", "mpg123", "mpeg", "audio", "player", "decoder")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "http://mpg123.org/"
+    license = "??"
```

Comment:
LGPL 2.1?
http://mpg123.org/cgi-bin/scm/mpg123/trunk/COPYING?view=markup
https://metadata.ftp-master.debian.org/changelogs//main/m/mpg123/mpg123_1.26.3-1_copyright

Reply:
Thanks for the review. I forgot to replace this placeholder.
It's LGPL-2.1-or-later and GPL-2.0-or-later btw (the file contains 2 licenses)